psi4/driver/procrouting/proc.py

```diff
@@ -1017,8 +1017,12 @@ def scf_wavefunction_factory(name, ref_wfn, reference):
         raise ValidationError("SCF: Unknown reference (%s) when building the Wavefunction." % reference)
 
     if disp_type:
-        wfn._disp_functor = empirical_dispersion.EmpericalDispersion(
-            disp_type[0], disp_type[1], tuple_params=modified_disp_params)
+        if type(disp_type) == type(dict()):
```

Comment:
Can you use `isinstance(disp_type, dict)` here?

Reply:
Done.